src/js/constants/deviceConstants.js

```diff
@@ -37,9 +39,47 @@ module.exports = {
   SET_DEVICE_LIMIT: 'SET_DEVICE_LIMIT',
 
   EXTERNAL_PROVIDER: {
-    amazon: { article: 'an', icon: AWS, title: 'AWS IoT core', provider: 'amazon', managementUrl: '/v1/aws-iot-manager/settings', enabled: false },
-    azure: { article: 'an', icon: Azure, title: 'Azure IoT Hub', provider: 'azure', managementUrl: '/v1/azure-iot-manager/settings', enabled: true },
-    google: { article: 'a', icon: GCP, title: 'Cloud IoT Core', provider: 'google', managementUrl: '/v1/gcp-iot-manager/settings', enabled: false }
+    amazon: {
+      article: 'an',
+      icon: AWS,
+      title: 'AWS IoT core',
+      provider: 'amazon',
+      managementUrl: '/v1/aws-iot-manager',
+      enabled: false,
+      configHint: <>For help finding your AWS IoT core connection string, check the AWS IoT documentation.</>
+    },
+    azure: {
+      article: 'an',
+      icon: Azure,
+      title: 'Azure IoT Hub',
+      provider: 'azure',
+      managementUrl: '/v1/azure-iot-manager',
```

Comment:
This must be renamed to "iot-manager" after latest changes to mendersoftware/integration